itou/prescribers/management/commands/update_refused_prescriber_organizations_kind.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from itou.prescribers.models import PrescriberOrganization
+
+
+class Command(BaseCommand):
+    """
+    The command updates "kind" value of PrescriberOrganizations with "refused" authorization status
+    """
+
+    help = "Update kind of prescriber_organization with REFUSED authorization status"
+
+    def show_counts(self, prescriber_orgs_refused):
+
+        self.stdout.write("Number of Prescriber Organizations with REFUSED authorization status :")
```

Comment:
```suggestion
        self.stdout.write("Number of Prescriber Organizations with REFUSED authorization status:")
```

Pinaillage : en anglais, il n'y a pas d'espace avant des les deux points (mais il y en a bien un après).